RecoHGCal/TICL/plugins/PatternRecognitionbyCAConstants.h

```diff
@@ -0,0 +1,15 @@
+#ifndef __RecoHGCal_TICL_PRbyCAConstants_H__
+#define __RecoHGCal_TICL_PRbyCAConstants_H__
+
+namespace ticl {
+  namespace patternbyCA {
+    constexpr int nEtaBins = 34;
+    constexpr int nPhiBins = 126;
+    constexpr int nLayers = 104;
+    typedef std::array<std::array<std::vector<unsigned int>, nEtaBins*nPhiBins>, nLayers>
+      tilePatternRecognitionByCA;
```

Comment:
typenames with upper case may be better.
Is there a good reason to maintain redundancy in the names? In the currently named namespace, this can be simply ```Tile``` or perhaps more descriptive ```LayerTiles```.